java-frontend/src/main/java/org/sonar/java/model/VisitorsBridgeForTests.java

```diff
@@ -58,7 +58,8 @@ public VisitorsBridgeForTests(Iterable<? extends JavaCheck> visitors, @Nullable
   }
 
   public VisitorsBridgeForTests(Iterable<? extends JavaCheck> visitors, List<File> projectClasspath, @Nullable SonarComponents sonarComponents) {
-    super(visitors, projectClasspath, sonarComponents, SymbolicExecutionMode.getMode(Iterables.toArray(visitors, JavaCheck.class)));
+    super(visitors, projectClasspath, sonarComponents, SymbolicExecutionMode.getMode(
```

Comment:
I think it might be simpler to write a new method in SymbolicExecutionMode accepting an Iterable instead of an array.